### DISCUSSION TITLE
Does brew uses system ruby?

### DISCUSSION THREAD

**Discussion by quackerex** [Homebrew/discussions#5254] [Everyday usage] [answered]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

Does adding `$(brew --prefix ruby)/bin` to `PATH` cause issues with homebrew?

**Comment by SMillerDev** [comment, marked as answer]
Brew uses its own ruby. Changing the one in the path won't affect brew
